app/Http/Controllers/FumaController.php

```diff
@@ -21,7 +21,7 @@
 class FumaController extends Controller
 {
 	public function appinfo(){
-		$app_config = parse_ini_file(storage_path()."/scripts/app.config", false, INI_SCANNER_RAW);
+		$app_config = parse_ini_file(scripts_path("app.config"), false, INI_SCANNER_RAW);
```

Comment:
I was just wondering if the file `app.config` should be under the `config` directory.

Reply:
Good question. The <root>/config directory is related to the Laravel framework i.e. the middleware and those config files are all php. 

This app.config config is not php and is directly relevant to the "business logic" of FUMA i.e. the python, R and perl scripts.  It is referenced in approx 37 script files in addition references settings in the 3 php job controllers that drive the FUMA jobs: S2G, G2F & Celltype.  

So because this is directly related to the jobs (business logic) I'd rather keep it separate from middleware config in <root>/config. Though there might be an argument for creating a config dir under scripts. Perhaps we can revisit this as a separate refactor.

Reply:
Sorry I should have been clearer. 
There is also a `config` directory under `scripts` (`scripts/config`) that houses the config for circos plots. 
I agree that this `app.config` is related to fuma-specific settings and not laravel.